source/_docs/automation.markdown

```diff
@@ -19,26 +19,26 @@ Before you can go ahead and create your own automations, it's important to learn
 
 The example consists of three different parts: a [trigger](/docs/automation/trigger/), a [condition](/docs/automation/condition/) and an [action](/docs/automation/action/).
 
-The first line is the **trigger** of the automation rule. Triggers describe events that should trigger the automation rule. In this case, it is a person arriving home, which can be observed in Home Assistant by observing the state of Paulus changing from 'not_home' to 'home'.
+The first line is the **trigger** of the automation rule. Triggers describe events that should start execution of the automation rule. In this case, it is a person arriving home, which can be observed in Home Assistant by Paulus' **state** going from 'not_home' to 'home'.
 
-The second line is the **condition**. Conditions are optional tests that can limit an automation rule to only work in your specific use cases. A condition will test against the current state of the system. This includes the current time, devices, people and other things like the sun. In this case, we only want to act when the sun has set.
+The second line is the **condition**. Conditions are optional tests that can limit an automation rule to only work in your specific use cases. A condition will test against the current **state** of an entity in the system. This includes the current time, devices, people and other things like the sun. In the example above, we only want to act when the sun has set.
```

Comment:
The original text says "[a] condition will test against the current state of the system". I felt this sentence might lead someone to understand that it is HA's state what is being queried, not values of "things" defined in HA. I used "entity" because [developer tools](https://www.home-assistant.io/docs/tools/dev-tools/) seems to put everything in that category (i.e., the pull down for 'entity' shows all the "things" available; thus, my noob mind understands that everything is called an "entity")

Reply:
It doesn't have to be a state, it could be trigger values, event data, or any other thing via custom templates as well. This changed makes the text untrue as entities are not the only thing here.